src/components/ui/ConfirmarEliminacion.js

```diff
@@ -33,12 +33,12 @@ export default function ConfirmarEliminacion({
       open={abrirModal}
     >
       <DialogTitle id="alert-dialog-slide-title">
-        ¿Confirma que desea eliminar <strong>{entidadAEliminar?.nombre}</strong>
-        ?
+        ¿Confirma que desea eliminar la actividad{' '}
+        <strong>{entidadAEliminar?.nombre}</strong>?
```

Comment:
Esto no va, porque el modal se usa en varias pantallas, no solo la de actividad.

```suggestion
        ¿Confirma que desea eliminar <strong>{entidadAEliminar?.nombre}</strong>
        ?
```